src/ddmd/backend/cgelem.c

```diff
@@ -2280,6 +2278,70 @@ STATIC elem * eldiv(elem *e, goal_t goal)
 
     if (OPTIMIZER)
     {
+        elem *e1 = e->E1;
```

Comment:
locals should start with lower case. Besides, it is called `e1` everywhere else in the compiler.